appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -110,6 +112,8 @@ internal class DebuggerViewModel(override val scope: AppcuesScope, debugMode: De
     val systemFonts: List<DebuggerFontItem>
         get() = debuggerFontManager.getSystemFonts()
 
+    val customComponents: Map<String, AppcuesCustomComponentView> = AppcuesCustomComponentDirectory.customComponents
```

Comment:
is there a way to use a delegating property here? so that if something gets added while the Debugger is running, it also shows up?